### PR TITLE
Feature/ufs 240 internal website listing

### PR DESCRIPTION
**Pull request by torchiad**

### CODE REVIEW

**Comment by torchiad** [comment]
@ascorbic I've got this one open but it's dependant on the other PR as I branched off of that as it has the model changes for the schema. 

What do you suggest to do with this? as it has this dependency https://github.com/urkiorg/ukripoc-opportunity/pull/9

Also I've tried to get rid of the last remaining any type but have failed so far...

File: src/theme.ts [hunk 1]
```diff
@@ -6,3 +6,21 @@ export const ukriGreen = "rgba(0, 130, 118, 1)";
...
+export const SettingsListItem = styled.div`
+    width: 100%;
+    height: 60px;
+    background-color: #d7d7d7;
+    line-height: 60px;
+    padding: 0 15px;
+    margin-bottom: 30px;
+`;
+
+export const FauxLink = styled.button`
```
Comment (ascorbic) [inline src/theme.ts]: You'll need to add a font here, and fix the vertical alignment

**Review by ascorbic — CHANGES_REQUESTED** [review]
Almost there! Just need to fix some styling on the fauxlink. I'd probably change the name to something more descriptive too, like LinkButton or something

File: src/theme.ts (hunk 1 above)
Comment (torchiad) [reply src/theme.ts]: I've made the changes mentioned, hoping that's the vertical alignment sorted as mentioned.

File: src/theme.ts [hunk 2]
```diff
@@ -16,11 +17,12 @@ export const SettingsListItem = styled.div`
...
-export const FauxLink = styled.button`
+export const LinkButton = styled.button`
+    font-family: ${NTA_LIGHT};
     text-decoration: underline;
     background-color: inherit;
     border: none;
     font-size: 20px;
-    color: blue;
-    line-height: 20px;
+    color: #005ea5;
```
Comment (ascorbic) [inline src/theme.ts]: Use a constant here

Comment (torchiad) [reply src/theme.ts]: Added

**Review by ascorbic — APPROVED** [review]

**Pull request merged**